src/main/java/com/google/firebase/auth/UserRecord.java

```diff
@@ -528,6 +532,17 @@ UpdateRequest setValidSince(long epochSeconds) {
       return this;
     }
 
+    UpdateRequest linkProvider(@NonNull UserProvider userProvider) {
```

Comment:
Note that this part of the api design is currently in flux, so it may change a bit yet. Sorry! :(

No action required (yet).